src/main/java/fr/insee/genesis/controller/utils/LoopIdentifier.java

```diff
@@ -15,6 +18,18 @@ private LoopIdentifier() {
 	public static String getLoopIdentifier(String variableName, VariablesMap variablesMap, int index) {
 		Variable variable = variablesMap.getVariable(variableName);
 		if (variable == null) {
+			if(variableName.startsWith(Constants.FILTER_RESULT_PREFIX)
+					&& variablesMap.hasVariable(variableName.replace(Constants.FILTER_RESULT_PREFIX,""))
+			){
+				Variable parentVariable = variablesMap.getVariable(variableName.replace(Constants.FILTER_RESULT_PREFIX,""));
```

Comment:
I don't understand the wording "parentvariable". It's not the parent as it's not a tree. It's more a "linkedvariable" or "attachedVariable"...

Reply:
Finally, we choose "relatedVariable", are you ok ?

Reply:
yes